spec/listing/listing_spec.js

```diff
@@ -1,97 +1,88 @@
-import { default as Backdraft } from "../../../src/entry.js";
+import List from "../../src/listing/list";
+import Item from "../../src/listing/item";
 
-describe("Listing Plugin", function() {
+import Collection from "../../src/collection";
 
-  var app;
-  var baseExports;
-  var collection;
+import { default as Backdraft } from "../../src/entry";
+
+class Sub extends Item {
+  get tagName() { return "li"; }
+
+  render() {
+    this.$el.html(this.model.get("name"));
+    return this;
+  }
+}
+
+class Main extends List {
+  get tagName() { return "ul"; }
+  get itemClass() { return Sub; }
+}
+
+describe("Listing", function() {
+  let collection;
 
   beforeEach(function() {
-    Backdraft.app.destroyAll();
-    app = Backdraft.app("myapp", {
-      plugins: [ "Listing" ]
-    });
-    app.model("M", {});
-    app.collection("col", {
-      model: app.Models.M
-    });
-    collection = new app.Collections.col();
-    baseExports = Backdraft.plugin("Base");
+    collection = new Collection();
   });
 
   describe("exports", function() {
-    var Listing;
-
-    beforeEach(function() {
-      Listing = Backdraft.plugin("Listing");
+    it("internal classes are available on Backdraft.Listing", function() {
+      expect(Backdraft.Listing.ListView).toEqual(List);
+      expect(Backdraft.Listing.ItemView).toEqual(Item);
     });
+  });
 
-    it("should expose an Item", function() {
-      app.view.listing.item("MyItem", {});
-      expect(new app.Views.MyItem()).toEqual(jasmine.any(Listing.Item));
-    });
+  describe("inheritance", function() {
+    describe("Item", function() {
+      it("is inheritable", function() {
+        class ChildItem extends Item {
 
-    it("should expose a List", function() {
-      app.view.listing("MyList", {});
-      expect(new app.Views.MyList({ collection: collection })).toEqual(jasmine.any(Listing.List));
+        }
+
+        expect(new ChildItem()).toEqual(jasmine.any(Item));
+      });
     });
-  });
 
-  describe("factories", function() {
+    describe("List", function() {
+      it("is inheritable", function() {
+        class ChildItem extends Item {
 
-    it("should expose #listing", function() {
-      app.view.listing("abc", {});
-      expect(new app.Views.abc({ collection: collection })).toEqual(jasmine.any(baseExports.View));
-    });
+        }
 
-    it("should expose #listing.item", function() {
-      app.view.listing.item("abc", {});
-      expect(new app.Views.abc).toEqual(jasmine.any(baseExports.View));
-    });
+        class ChildList extends List {
+          get itemClass() {
+            return ChildItem;
+          }
+        }
 
-    it("requires that a collection be provided", function() {
-      expect(function() {
-        app.view.listing("abc", {});
-        new app.Views.abc();
-      }).toThrowError("A collection must be provided");
-    });
+        expect(new ChildItem()).toEqual(jasmine.any(Item));
+        expect(new ChildList({collection})).toEqual(jasmine.any(List));
```

Comment:
Need spaces padding inside of `{}` like you've got on line 79. Not sure why ESL didn't pick this up.

Reply:
I'll look into updating our EsLint config.